test/watchCases/simple/multi-compiler/webpack.config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = [{
+    name: "changing",
+    entry: "./index.js",
+    output: {
+        filename: "./bundle.js"
+    }
+}, {
+    name: "static",
+    entry: "./static-file.js",
+    output: {
+        filename: "./static.js"
+    }
+}];
```

Comment:
That wouldn't work.
I mean that currently (including before my changes), only the first MultiStats is really batched, after that, `if(compilerStatus.every(Boolean))` is always satisfied and each of the stats is forwarded to handler independently.
Here https://github.com/uncleyo/webpack/commit/d4a7aac61c5cd43d9e80782ffaab1044e546536d is what I tried and it doesn't work.
Compilations do not share watchers, so I guess you'd need to aggregate them somehow.
But I think it's fine if these stats are sent to handler one by one in watch mode, don't you?